### PR TITLE
Updated Changelog to reflect recent changes

### PR DESCRIPTION
**Pull request by jhdalek55**
Added all the significant changes made to the Standard during our proofreading sessions

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -41,3 +41,23 @@ The following is a brief summary of those edits:
 - Specified in the procedure for checking hashes that the ECU SHOULD check that the length of the image matches the length listed in the metadata.
 
 - Modified wording to make verifying a time message optional if the ECU does not have the capacity to verify a time message.
+
+ 
+* Formally changed the name of our deployment considerations document  to *Uptane Deployment Best Practices.* This was done to better reflect naming conventions within the community.
+ 
+* Corrected additional capitalization and punctuation usages for consistency, including imposing the consistent use of the Oxford comma in a series of items within a sentence, and placing a comma after e.g. and i.e.
+ 
+* Corrected other stylistic/formatting issues that interfered with clarity, such as extraneous commas and use of whitespace,
```
Comment (patrickvacek) [inline changelog.md:50]:
```suggestion
* Corrected other stylistic/formatting issues that interfered with clarity, such as extraneous commas and use of whitespace.
```
:)

File: changelog.md [hunk 2]
```diff
@@ -41,3 +41,23 @@ The following is a brief summary of those edits:
...
 - Modified wording to make verifying a time message optional if the ECU does not have the capacity to verify a time message.
+
+ 
+* Formally changed the name of our deployment considerations document  to *Uptane Deployment Best Practices.* This was done to better reflect naming conventions within the community.
+ 
+* Corrected additional capitalization and punctuation usages for consistency, including imposing the consistent use of the Oxford comma in a series of items within a sentence, and placing a comma after e.g. and i.e.
+ 
+* Corrected other stylistic/formatting issues that interfered with clarity, such as extraneous commas and use of whitespace, 
+ 
+* Remove the use of the phrase “best practices” when not referring to the *Uptane Deployment Best Practices.*
```
Comment (patrickvacek) [inline changelog.md:52]:
```suggestion
* Removed the use of the phrase “best practices” when not referring to the *Uptane Deployment Best Practices.*
```

**Review by patrickvacek — COMMENTED** [review]
Looks about right to me, thanks! Does it matter that the previous items use `-` but the new ones `*`? It seems like it doesn't really.

**Comment by jhdalek55** [comment]
I created the added links offline and just picked the other option for creating bullets. I was going to change them, but it didn't seem to make a difference in terms of how things looked. I may go in and reverse them anyway, just for consistency. 
I'm going to try to finish up proofing of the Deployment pages later today or tomorrow. I have four more sections to go through (key management, normal ops, security, and repository). By the end of the week I also plan to finish proofing the Standard. I'm starting from 5.4.4.3.

**Comment by jhdalek55** [comment]
I will leave this open until we resolve Issue #189 on the Standard, as we will need to log that change in as well.

**Pull request merged**